src/contracts/core/AllocationManager.sol

```diff
@@ -200,7 +200,9 @@ contract AllocationManager is
                         // the deallocation delay. This magnitude remains slashable until then.
                         deallocationQueue[operator][strategy].pushBack(operatorSet.key());
 
-                        allocation.effectBlock = uint32(block.number) + DEALLOCATION_DELAY;
+                        // deallocations are slashable in the window [block.number, block.number + deallocationDelay]
+                        // therefore, the effectBlock is set to the block right after the slashable window
+                        allocation.effectBlock = uint32(block.number) + DEALLOCATION_DELAY + 1;
```

Comment:
Note: every comparison to `effectBlock` is `< effectBlock`. There is some minor difference in naming (and resulting comparisons) because we do:

- < effectBlock
- <= slashableUntil 

Nomenclature does make sense here though